controller/cache/info.go

```diff
@@ -146,6 +146,15 @@ func populateIngressInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 						tlshost := tlsline["host"]
 						if tlshost == host {
 							stringPort = "https"
+							continue
+						}
+						if hosts := tlsline["hosts"]; hosts != nil {
+							tlshosts := tlsline["hosts"].(map[string]interface{})
```

Comment:
It is unlikely but if one day `hosts` got converted to list this code will crash. Can you add a cast check similar to check of `tlsline`? E.g.:

```
tlshosts, ok := tlsline["hosts"].(map[string]interface{})
if ok {
	for j := range tlshosts {
		if tlshosts[j] == host {
			stringPort = "https"
		}
	}
}
```